projects/observability/src/shared/components/explore-filter-link/explore-filter-link.component.ts

```diff
@@ -0,0 +1,20 @@
+import { ChangeDetectionStrategy, Component, Input } from '@angular/core';
+import { IconType } from '@hypertrace/assets-library';
+import { NavigationParams } from '@hypertrace/common';
+import { IconSize } from '@hypertrace/components';
+
+@Component({
+  selector: 'ht-explore-filter-link',
+  styleUrls: ['./explore-filter-link.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  template: `
+    <ht-link class="explore-filter" [paramsOrUrl]="this.paramsOrUrl">
+      <ng-content></ng-content>
+      <ht-icon class="filter-icon" size="${IconSize.ExtraSmall}" icon="${IconType.Filter}"></ht-icon>
+    </ht-link>
+  `
+})
+export class ExploreFilterLinkComponent {
```

Comment:
Feels a bit weird to call this explore filter link when it doesn't reference explorer, it just adds a filter icon around a provided link + content.

Reply:
The icon type should be more specific to explorer.